paper.md

```diff
@@ -0,0 +1,144 @@
+---
+title: 'CMinx: A CMake Documentation Generator'
+tags:
+  - Python
+  - CMake
+  - Documentation
+authors:
+  - name: Branden Butler
+    equal-contrib: true
+    affiliation: "1, 2"
+  - name: Ryan M. Richard
+    orcid: 0000-0003-4235-5179
+    equal-contrib: false
+    corresponding: true # (This is how to denote the corresponding author)
+    affiliation: "1, 2"
+affiliations:
+ - name: Ames National Laboratory, Ames, IA, USA
+   index: 1
+ - name: Iowa State University, Ames, IA, USA
+   index: 2
+date: August 2022
+bibliography: paper.bib
+---
+
+# Summary
+
+This manuscript introduces CMinx, a program for generating
+application programming interface (API) documentation for CMake modules
+(code written in the CMake language). Since most of CMinx's intended audience
+is comprised of C/C++ developers, CMinx is designed to operate similar to
+Doxygen [@doxygen] (the *de facto* C/C++ API documentation tool).
+Specifically, developers annotate their CMake source with documentation
+comments (traditional CMake block comments, starting with an extra "`[`"
+character). The documentation comments, written in reST [@rest],
+describe to the reader how the functions, parameters, and variables should be
+used. Running CMinx on the annotated source code generates reST files containing
+the API documentation. The reST files can then be converted into static
+websites with tools such as Sphinx [@sphinx] or
+easily converted to another format via Pandoc [@pandoc].
+
+Unlike other solutions for documenting CMake modules, CMinx knows the CMake
+language's grammar. This enables CMinx to automatically extract function/macro
+signatures (even when functions are not documented). CMinx also integrates
+seamlessly into existing CMake build systems. CMinx's output is highly
+customizable and easily controlled via a YAML [@yaml] configuration file.
```

Comment:
I can find this claim nowhere in the documentation. How can CMinx be customized with a configuration file? Please clarify or remove the claim.

Reply:
This is a relatively new feature. At the moment I think it's only documented in the ``cminx --help`` message. I'll make sure that gets into the documentation.